web/src/components/Callout.tsx

```diff
@@ -0,0 +1,77 @@
+import { ReactElement } from "react";
+
+export type CalloutTypes = "informational" | "conditional" | "warning" | "note";
+
+interface Props {
+  calloutType: CalloutTypes;
+  header?: string;
+  children: string;
+  icon?: "true" | "false";
+}
+
+interface CalloutMappingObj {
+  containerStyling: string;
+  headingStyling: string;
+  headingDefaultText: string;
+  bodyStyling: string;
+  iconStyling: string;
+}
+
+export const calloutComponentMapping: Record<string, CalloutMappingObj> = {
+  informational: {
+    containerStyling: "bg-accent-cool-lightest",
+    headingStyling: "text-accent-cool-more-dark",
+    headingDefaultText: "Keep in mind:",
+    bodyStyling: "text-primary-darker",
+    iconStyling: "callout-informational-icon",
+  },
+  conditional: {
+    containerStyling: "bg-primary-extra-light",
+    headingStyling: "text-primary-darker",
+    headingDefaultText: "Once completed, you will have...",
+    bodyStyling: "text-primary-darker",
+    iconStyling: "callout-conditional-icon",
+  },
+  warning: {
+    containerStyling: "bg-warning-extra-light",
+    headingStyling: "text-accent-warm-darker",
+    headingDefaultText: "Be advised:",
+    bodyStyling: "text-primary-darker",
+    iconStyling: "callout-warning-icon",
+  },
+  note: {
+    containerStyling: "bg-base-lightest",
+    headingStyling: "text-base-darkest",
+    headingDefaultText: "Note:",
+    bodyStyling: "text-primary-darker",
```

Comment:
[sand] All of these instances of `bodyStyling` are the same. Should we just add `text-primary-darker` directly to the element?